src/io-methods/filesystem.js

```diff
@@ -1,47 +1,17 @@
 "use strict";
-const fs = require("fs");
-const config = require("../config").config;
+const fs = require("fs/promises");
+const { getConfig } = require("../config");
 const jsHelpers = require("./js-file-helpers.js");
 
 exports.readManifest = function (filePath) {
-  return new Promise((resolve, reject) => {
-    //create file if not already created
-    fs.open(filePath, "a", function (err, fd) {
```

Comment:
No longer create the file if it doesn't exist. Instead, throw an error if it doesn't exist.